src/common/marlin_client.c

```diff
@@ -154,24 +155,32 @@ int marlin_client_id(void) {
     return 0;
 }
 
-int marlin_client_set_dialog_open_cb(dialog_open_cb_t cb) {
+int marlin_client_set_fsm_create_cb(fsm_create_t cb) {
```

Comment:
I have no clue what cb is. Return value is not documented.

Reply:
callback

Reply:
commented